sdk/js-query/src/mock/index.ts

```diff
@@ -485,6 +490,113 @@ export class QueryProxyMock {
             )
           )
         );
+      } else if (type === ChainQueryType.SolanaPda) {
+        const query = perChainRequest.query as SolanaPdaQueryRequest;
+        // Validate the request and convert the PDAs into accounts.
+        if (query.commitment !== "finalized") {
+          throw new Error(
+            `Invalid commitment in sol_account query request, must be "finalized"`
+          );
+        }
+        if (
+          query.dataSliceLength === BigInt(0) &&
+          query.dataSliceOffset !== BigInt(0)
+        ) {
+          throw new Error(
+            `data slice offset may not be set if data slice length is zero`
+          );
+        }
+        if (query.pdas.length <= 0) {
+          throw new Error(`does not contain any account entries`);
+        }
+        if (query.pdas.length > 255) {
+          throw new Error(`too many account entries`);
+        }
+
+        let accounts: string[] = [];
+        let bumps: number[] = [];
+        query.pdas.forEach((pda) => {
+          if (pda.programAddress.length != 32) {
+            throw new Error(`invalid program address length`);
+          }
+
+          const [acct, bump] = PublicKey.findProgramAddressSync(
+            pda.seeds,
+            new PublicKey(pda.programAddress)
+          );
+          accounts.push(acct.toString());
+          bumps.push(bump);
+        });
+
+        let opts: SolanaGetMultipleAccountsOpts = {
+          commitment: query.commitment,
+        };
+        if (query.minContextSlot != BigInt(0)) {
+          opts.minContextSlot = Number(query.minContextSlot);
```

Comment:
I wonder if opts.minContextSlot should be `bigint` instead of `number` just to be consistent

Reply:
I don't think so, as it has to be serialized to a number field in the JSON. https://solana.com/docs/rpc/http/getaccountinfo